pkg/translator/loki/convert_test.go

```diff
@@ -59,6 +59,19 @@ func TestConvertAttributesAndMerge(t *testing.T) {
 				"host.name": "guarana",
 			},
 		},
+		{
+			desc:     "selected attributes from resource attributes",
```

Comment:
could you please add to the desc `"should be included"` at the end? So it is more precise
`"selected attributes from resource attributes should be included"`

Reply:
updated